src/screens/ProfileScreen.js

```diff
@@ -253,11 +258,26 @@ export default class ProfileScreen extends React.Component {
     }
   }
 
+  navigateEditScreen() {
+    const params = this.getParams();
+    return(
+    <TouchableOpacity
+          onPress={() => this.props.navigation.navigate('EditProfileScreen', {
```

Comment:
you'll need to create a stack navigator for the profile screen in order for this to work I think. Look in the `src/navigators` files to see an example of how this is done. Currently the home screen also needs to display the profile screen sometimes so that's why we have a `HomeStackNavigator`